packages/spec-lib/src/lib.ts

```diff
@@ -0,0 +1,30 @@
+import { createTypeSpecLibrary, paramMessage } from "@typespec/compiler";
+
+const repoUrl = `https://github.com/Azure/cadl-ranch`;
```

Comment:
I think this will fail you might have to not use a variable due to how diagnostic lib work and detect interpolation

Reply:
@timotheeguerin I have updated the repoUrl to `https://github.com/Microsoft/typespec`. Would that be fine?

Reply:
no i mean the interpolation doesn't work with `paramMessage` as it will expect a key called `https://github.com/Microsoft/typespec`

Reply:
check the current state of lib.ts in the cadl-ranch repo

Reply:
Thanks for the explanation. Resolved.